documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,405 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+Migrate virtual machines (VMs) from VMware vSphere, {rhv-full}, or {osp} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, known issues, and resolved issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Migration from OpenStack moves to being a fully supported feature
+
+In this version of {project-short}, migration using {osp} source providers graduated from a Technology Preview feature to a fully supported feature.
+
+.Integration of the create and update provider user interface
+
+The user interface of the source providers and destination providers now aligns with the look and feel of the {ocp} web console and displays the current data.
+
+.Standalone UI
+
+The old UI of {project-short} 2.3 cannot run by setting `feature_ui: true` in ForkliftController anymore.
+
+.Improved errors logged in populator pods
+
+In earlier releases of {project-short} {project-version}, populator pods were always restarted on failure. This made gathering logs from the failed pods difficult. {project-short} 2.5.3 has a maximum number of three restarts of populator pods. On the third and final time, the populator pod remains in the fail status, and its logs can then be easily gathered by `must-gather` and by `forklift-controller`, to know this step has failed. link:https://issues.redhat.com/browse/MTV-818[(MTV-818)]
+
+.The `VirtualMachineClusterFlavor` CRDs replace {ocp-name} templates
```

Comment:
there's a bit of confusion here - we never added flavors since they were replaced with instance type and instance type preferences
in MTV 2.6 the migrated VMs are based on instance type preferences as a first priority and, as you mention below, we fall back to OpenShift ttemplates when a relevant preference is not available.
cc @liranr23

Reply:
yup. there is no flavor. we use `VirtualMachineInstancetype` (namespaced, user created) and if we don't find it, we try `VirtualMachineClusterInstancetype` (cluster-wide, usually CNV created, but user with permissions can create this kind as well).

Reply:
changed text